ERROR - 2024-10-22 15:56:11 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 15:56:11 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 15:56:28 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 15:56:28 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 15:57:33 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 15:59:20 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 15:59:34 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 15:59:34 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:00:25 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:00:25 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:01:41 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:01:41 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:03:57 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:03:57 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:05:23 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:05:23 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:08:00 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:08:00 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(956): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:08:58 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:09:20 --> mysqli_sql_exception: Operand should contain 1 column(s) in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `detailc...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `detailc...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `detailc...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2475): CodeIgniter\Database\BaseConnection->query('UPDATE `detailc...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(385): CodeIgniter\Database\BaseBuilder->update()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(949): CodeIgniter\Model->doUpdate(Array, Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(776): CodeIgniter\BaseModel->update(Array, Array)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(945): CodeIgniter\Model->update('62', Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:09:21 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(957): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:11:24 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(953): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:19:14 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 E:\Wamp64\www\Assurance\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(955): CodeIgniter\BaseModel->save(Array)
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#11 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
ERROR - 2024-10-22 16:20:09 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(955): CodeIgniter\Model->insert(Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:21:39 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(955): CodeIgniter\Model->insert(Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:21:59 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(954): CodeIgniter\Model->insert(Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:24:26 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(955): CodeIgniter\Model->insert(Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
ERROR - 2024-10-22 16:24:56 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 E:\Wamp64\www\Assurance\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `so...', 0)
#1 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `so...')
#2 E:\Wamp64\www\Assurance\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `so...')
#3 E:\Wamp64\www\Assurance\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `so...', Array, false)
#4 E:\Wamp64\www\Assurance\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 E:\Wamp64\www\Assurance\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 E:\Wamp64\www\Assurance\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 E:\Wamp64\www\Assurance\app\Controllers\Consultation\ConsultationCont.php(953): CodeIgniter\Model->insert(Array)
#8 E:\Wamp64\www\Assurance\system\CodeIgniter.php(942): App\Controllers\consultation\ConsultationCont->add_autreacte()
#9 E:\Wamp64\www\Assurance\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\consultation\ConsultationCont))
#10 E:\Wamp64\www\Assurance\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 E:\Wamp64\www\Assurance\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
